### PR TITLE
solution docs page ui exlm-889

### PR DESCRIPTION
**Pull request by nitin-rachabathuni**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-889

Test URLs:

- Before: https://experienceleague.adobe.com/docs/?lang=en 
- After: https://feature-exlm-889--exlm--adobe-experience-league.hlx.page/en/docs

### CODE REVIEW

File: blocks/cloud-solutions/cloud-solutions.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    });
+    container.appendChild(listWrapper);
+    const target = wrapperEl.querySelector(':scope > h2');
+    if (target) {
+      wrapperEl.insertBefore(container, target);
+    } else {
+      wrapperEl.appendChild(container);
+    }
+  }
+}
```
Comment (noormd08) [inline blocks/cloud-solutions/cloud-solutions.js:39]: As discussed, cleanup the above code with single h2 element and llop through based on h3.

File: blocks/cloud-solutions/cloud-solutions.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+@media (min-width: 900px) {
+    .cloud-solutions-list-container {
+        width: 30%;
+    }
+
+    .cloud-solutions-list-wrapper {
+        gap: 8px;
+    }
+}
```
Comment (noormd08) [inline blocks/cloud-solutions/cloud-solutions.css:75]: Remove unwanted CSS

File: blocks/cloud-solutions/cloud-solutions.js (hunk 1 above)
Comment (nitin-rachabathuni) [reply blocks/cloud-solutions/cloud-solutions.js:32]: done @noormd08

File: blocks/cloud-solutions/cloud-solutions.css (hunk 2 above)
Comment (nitin-rachabathuni) [reply blocks/cloud-solutions/cloud-solutions.css:75]: done @noormd08 ,

File: .eslintrc.cjs [hunk 3]
```diff
@@ -13,7 +13,7 @@ module.exports = {
   rules: {
     // allow reassigning param
     'no-param-reassign': [2, { props: false }],
-    'linebreak-style': ['error', 'unix'],
+    'linebreak-style': 0, //['error', 'unix'],
```
Comment (noormd08) [inline .eslintrc.cjs:16]: Don't make any changes on eslint

**Review by noormd08 — DISMISSED** [review]

**Review by noormd08 — APPROVED** [review]

**Pull request merged**